test/domains/operators/countTypeMismatch.chpl

```diff
@@ -0,0 +1,17 @@
+const n = 10: uint(8);
+
+var D1 = {1..n};
+var D2 = {1..n, 1..n};
+
+var A1: [D1] int = [i in D1] i;
+var A2: [D2] int = [(i,j) in D2] i*10 + j;
+
+config const c = 3, c2 = 4;
+
+writeln(D1#c);
+//writeln(D2#c);
```

Comment:
Was commenting this and the other line below intentionally?  It looks like the .good file still expects them

Reply:
Yeah, it was intentional—the count operator currently doesn't accept a scalar value for multidimensional domains and arrays, but requires a tuple (which is what I had hoped, but wasn't sure of).  I could've/maybe should've removed the commented out lines, but kept them in to give a gentle indication that I'd tried them but that they're not worth testing (in this test anyway).  I think the .good file is correct, where:

* line 11 of the source corresponds to line 1
* line 13 maps to line 2
* line 15 maps to line 3
* line 17 maps to lines 4-6

Reply:
Ahhhh, that makes sense.  I'd either remove the lines or give a little more indication of why they aren't in use, in case someone looks at the test later and wonders why they're commented out